From ea8c585cf0cd0b4bd72ddc8755c13d27a669bea8 Mon Sep 17 00:00:00 2001
From: Diana Craciun <diana.craciun@nxp.com>
Date: Mon, 3 Jun 2019 15:06:07 +0300
Subject: [PATCH 523/741] Changed VFIO_DEVICE_FLAGS_FSL_MC define

commit db630d7a07cf9c18230f9e8531ea4f0ca330ba6a from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

Newer versions of kernel have more devices. In order to use the same QEMU
for all versions, we need to update the define.

Signed-off-by: Diana Craciun <diana.craciun@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 include/uapi/linux/vfio.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/uapi/linux/vfio.h b/include/uapi/linux/vfio.h
index 0297140199d1..e6ae031b71b6 100644
--- a/include/uapi/linux/vfio.h
+++ b/include/uapi/linux/vfio.h
@@ -200,8 +200,8 @@ struct vfio_device_info {
 #define VFIO_DEVICE_FLAGS_PLATFORM (1 << 2)	/* vfio-platform device */
 #define VFIO_DEVICE_FLAGS_AMBA  (1 << 3)	/* vfio-amba device */
 #define VFIO_DEVICE_FLAGS_CCW	(1 << 4)	/* vfio-ccw device */
-#define VFIO_DEVICE_FLAGS_FSL_MC (1 << 5)      /* vfio-fsl-mc device */
-#define VFIO_DEVICE_FLAGS_AP	(1 << 6)	/* vfio-ap device */
+#define VFIO_DEVICE_FLAGS_AP	(1 << 5)	/* vfio-ap device */
+#define VFIO_DEVICE_FLAGS_FSL_MC (1 << 6)      /* vfio-fsl-mc device */
 	__u32	num_regions;	/* Max region index + 1 */
 	__u32	num_irqs;	/* Max IRQ index + 1 */
 };
-- 
2.17.1

